unitTests/ui/DatePickerTest.ts

```diff
@@ -66,5 +66,17 @@ export function DatePickerTest() {
         expect(spy).toHaveBeenCalledWith(picker);
       });
     });
+
+    describe('build', () => {
```

Comment:
Rather than wrapping in a `describe` and increasing the nesting one level, what about adding this to the description of the it? e.g.
```
when calling #build, it returns the element on which the date picker is bound
```

Same comment for the describe on line 76.

Reply:
Yes it's not a wrong way of doing it, but it wouldn't be consistent with our other UTs